src/test/java/io/getunleash/DefaultUnleashTest.java

```diff
@@ -310,6 +310,7 @@ public void asynchronous_fetch_on_initialisation_fails_silently_and_retries()
                         .build();
 
         Unleash unleash = new DefaultUnleash(config);
+        Thread.sleep(1);
```

Comment:
*cries in Windows* So this is the failing test that sent me down the rabbit hole of the client spec tests (which aren't yet fixed). It looks like there's a very subtle difference in the thread scheduler between Windows and Linux. Linux is more precise, which means that the fetch has always completed before the verify runs on the next line. On Windows, this never completes without this sleep

Reply:
Yes, we can't wait on DefaultUnleash constructor, but tbh, I don't like it does `fetchFeatures` as part of constructing the object... I think fetch features should be part of some `initialize()` method and then maybe you'd be able to wait on it... but sleeping is fine, people need to sleep :)

Reply:
Agreed, I'd like it to change too and what you propose is what I'd like to do too, Constructors should construct, not spawn threads.

Small steps though, this is legacy behaviour that's being surfaced because we're doing Windows tests now. Can fix next series of PRs